test/helpers/common.ts

```diff
@@ -12,6 +12,11 @@ export async function expectToFail(fun: () => any, message: string) {
   ).to.be.rejectedWith(message);
 }
 
+export function expectEvent(receipt: ContractReceipt, expectedEventName: string) {
```

Comment:
Maybe we should have this function receive also the event's parameters so we could test they are correct?

Reply:
I did for most cases. The remaining 2 are hard to calculate off-chain. Let me know if you think it's enough